### ISSUE TITLE
[BUG] 2.1.2 wheels fail to use `_GLIBCXX_USE_CXX11_ABI=1`

### ISSUE THREAD

**Issue by njzjz** [deepmodeling/deepmd-kit#1791]
### Bug summary

TF 2.9 has used `_GLIBCXX_USE_CXX11_ABI=1`. But 2.1.2 wheels, which were built against TF 2.9, applied the flag `_GLIBCXX_USE_CXX11_ABI=1` but still had old symbols, causing errors.

Per https://stackoverflow.com/a/50850388/9567349 and https://bugzilla.redhat.com/show_bug.cgi?id=1546704, this flag does not work with `devtoolset` on RHEL6 and RHEL7. `manylinux2014` uses CentOS 7. We may need to bump to a newer image...

### DeePMD-kit Version

2.1.2

### TensorFlow Version

2.9.1

### How did you download the software?

pip

### Input Files, Running Commands, Error Log, etc.

> tensorflow.python.framework.errors_impl.NotFoundError: /opt/_internal/cpython-3.10.5/lib/python3.10/site-packages/deepmd/op/libop_abi.so: undefined symbol:
> _ZNK10tensorflow8OpKernel11TraceStringERKNS_15OpKernelContextEb

```
c++filt _ZNK10tensorflow8OpKernel11TraceStringERKNS_15OpKernelContextEb
```
> tensorflow::OpKernel::TraceString(tensorflow::OpKernelContext const&, bool) const

Symbols in TF library:

```
c++filt _ZNK10tensorflow8OpKernel11TraceStringB5cxx11ERKNS_15OpKernelContextEb
```
> tensorflow::OpKernel::TraceString[abi:cxx11](tensorflow::OpKernelContext const&, bool) const

### Steps to Reproduce

```sh
python -m pip install tensorflow==2.9.1
python -m pip install -v .
python -m deepmd -h
```

or install wheels from pypi:
```sh
python -m pip install tensorflow==2.9.1 deepmd-kit=2.1.2
```

**Comment by njzjz** [comment]
I've removed all 2.1.2 wheels on the PyPi.
